src/main/groovy/life/qbic/samplestatus/reporter/services/utils/SampleStatusMapper.groovy

```diff
@@ -2,53 +2,53 @@ package life.qbic.samplestatus.reporter.services.utils
 
 import life.qbic.samplestatus.reporter.Result
 
-import java.util.function.Function
-
 /**
  * <b>Class SampleStatusMapper<b>
  *
  * <p>Takes a String value and tries to map it to a known sample status.</p>
  *
  * @since 1.0.0
  */
-class SampleStatusMapper implements Function<String, Result<String, Exception>> {
+class SampleStatusMapper {
 
-  enum SampleStatus {
-    SAMPLE_RECEIVED("Sample received", "SAMPLE_RECEIVED"),
-    SAMPLE_QC_PASSED("QC passed", "SAMPLE_QC_PASS"),
-    SAMPLE_QC_FAILED("QC failed", "SAMPLE_QC_FAIL"),
-    LIBRARY_PREP_FINISHED("Library completed", "LIBRARY_PREP_FINISHED")
+  enum KnownSampleStatus {
+    SAMPLE_RECEIVED("Sample received", "SAMPLE_RECEIVED", false),
+    SAMPLE_QC_PASSED("QC passed", "SAMPLE_QC_PASS", false),
+    SAMPLE_QC_FAILED("QC failed", "SAMPLE_QC_FAIL", false),
+    LIBRARY_PREP_FINISHED("Library completed", "LIBRARY_PREP_FINISHED", false),
+    SEQUENCING_COMPLETED("Sequencing completed", "", true)
 
 
     private final String limsStatus
     private final String qbicStatus
+    private final boolean ignored
```

Comment:
NIT you could also name it 'active' and mark all statuses that are activly used with true